draft-irtf-cfrg-rsa-blind-signatures.md

```diff
@@ -540,7 +569,7 @@ The following parameters are specified:
   encoded as hexadecimal strings.
 - sig: The output message signature.
 
-Test vector for probabilistic signatures (sLen=48):
+Test vector for probabilistic signatures (sLen=48, random messages, no message salting):
```

Comment:
A TODO is also to add test vectors with the additional hashing step.

Reply:
I have an update to the Python code that generates the test vectors, that I can commit right after this PR got merged.